node_modules/gh-admins/lib/rest.js

```diff
@@ -53,7 +55,13 @@ GrassHopper.globalAdminRouter.on('get', '/api/admins', function(req, res) {
  * @Return      {Me}                                            The current global administrator
  */
 GrassHopper.globalAdminRouter.on('get', '/api/me', function(req, res) {
-    res.status(200).send(req.ctx.user);
+    var me = {
```

Comment:
It's kinda fugly to write a generic `getMe` function and use it post-login. There's no context available yet at that time. There wouldn't be much benefit to it as the me feed only adds the `anon` key right now.